RetroBar/PropertiesWindow.xaml

```diff
@@ -310,6 +315,22 @@
                             </DockPanel>
                         </StackPanel>
                     </GroupBox>
+                    <GroupBox Header="{DynamicResource taskbar_scale}">
+                        <DockPanel Margin="20,0">
+                            <Label DockPanel.Dock="Left" Content="{DynamicResource taskbar_scale_1x}" />
+                            <Label DockPanel.Dock="Right" Content="{DynamicResource taskbar_scale_2x}" />
+                            <Slider Name="sldTaskbarScale"
+                                    Orientation="Horizontal"
+                                    IsSnapToTickEnabled="True"
+                                    Value="{Binding Source={x:Static Settings:Settings.Instance}, Path=TaskbarScale, UpdateSourceTrigger=PropertyChanged}"
+                                    Minimum="1"
+                                    Maximum="2"
+                                    AutoToolTipPlacement="BottomRight"
+                                    AutoToolTipPrecision="2"
+                                    TickFrequency="0.25"
+                                    TickPlacement="BottomRight" />
+                        </DockPanel>
+                    </GroupBox>
```

Comment:
For fixed values, it would be better to use a `ComboBox` with the values in percent:
![cboPer](https://user-images.githubusercontent.com/28744392/236619375-8a531d7e-5e06-4bb9-b743-8cbc7149fbff.png)

Or, if we still want to use the slider, we should be able to offer more options, perhaps by changing the `TickFrequency` to 0.05 we could give users more options:
![sldPer](https://user-images.githubusercontent.com/28744392/236619722-cd51d9f7-7b95-478b-b92d-f135389bedea.png)


Finally, if none of this could be better, we would use a numeric text box where the user could set a value from a minimum of 50% or 100% to a maximum of 200%. But that might be overkill.

Reply:
Thanks for the good ideas! I originally chose 0.25 increment because that is where the themes look best, however it doesn't hurt to provide more fine-grained control. I also agree with using percentage for display. I'll get this pushed once my network returns.
![Screenshot 2023-05-06 at 11 25 23 AM](https://user-images.githubusercontent.com/7373633/236635780-53f24d68-e1de-4622-bdf5-d8364036a6b8.png)